Cargo.toml

```diff
@@ -142,6 +141,7 @@ log_wrappers = { path = "components/log_wrappers" }
 engine = { path = "components/engine" }
 tikv_util = { path = "components/tikv_util" }
 farmhash = "1.1.5"
+fxhash = "0.2.1"
```

Comment:
em, why do we add fxhash again? I remember we remove it before.

Reply:
The standard implementation of HashMap still uses `SipHash 1-3` algorithm whereas the hashbrown used the `fxHash` algorithm. I had to add the `fxhash` dependency to use the same algorithm. I was initially hesitant while adding this dependency because I don't think it is maintained now.

Reply:
There's also [discussion](https://internals.rust-lang.org/t/a-new-default-hasher-for-hashmap/9747) of using [ahash](https://docs.rs/ahash/0.2.0/ahash/) to replace SipHash in std. We could try that if maintenance is a problem.

Reply:
`ahash` seems like a good option. Maybe we can wait for the PR [hashbrown/97](https://github.com/rust-lang/hashbrown/pull/97) to land? There are still working on optimisations and hopefully it will become the default hasher soon. Till then, we can continue to use fxhash.